internal/api/impl/proxy/proxy.go

```diff
@@ -406,14 +406,14 @@ func (h *httpProxy) serve(c echo.Context) error {
 
 	// redirect the request to the datasource
 	req.URL.Path = h.path
-	logrus.Debugf("request will be redirected to %q", h.config.URL.String())
+	logrus.Debugf("request will be redirected to %q", h.config.URL.URL.String())
 
 	// Set up the proxy
 	var proxyErr error
-	reverseProxy := httputil.NewSingleHostReverseProxy(h.config.URL)
+	reverseProxy := httputil.NewSingleHostReverseProxy(h.config.URL.URL)
 	reverseProxy.ErrorHandler = func(writer http.ResponseWriter, request *http.Request, err error) {
-		desc := h.config.URL.String()
```

Comment:
Do you really need these .URL ? I think that .String() is accessible anyway from the parent.